### ISSUE TITLE
Support providing local (client) certificate file including private key

### ISSUE THREAD

**Issue by jgehrcke** [cannatag/ldap3#203]
Python's SSL module supports presenting a client certificate via `load_cert_chain()`. Relevant docs:

https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_cert_chain

Note how `keyfile` is not _required_.

About that: https://docs.python.org/3/library/ssl.html#combined-key-and-certificate

Quote:
```
Often the private key is stored in the same file as the certificate; in this case,
only the certfile parameter to SSLContext.load_cert_chain() and
wrap_socket() needs to be passed. If the private key is stored with the
certificate, it should come before the first certificate in the certificate chain:

-----BEGIN RSA PRIVATE KEY-----
... (private key in base64 encoding) ...
-----END RSA PRIVATE KEY-----
-----BEGIN CERTIFICATE-----
... (certificate in base64 PEM encoding) ...
-----END CERTIFICATE-----
```

ldap3 on the one hand uses `load_cert_chain()` and exposes the two main arguments to the user of ldap3, but it requires both of them (only invokes `load_cert_chain()` if a key file was provided):

https://github.com/cannatag/ldap3/blob/b5abdee3fe38c56950a782cd8d29e59631aae967/ldap3/core/tls.py#L177

It's quite convenient to have key and cert live in the same file. We should change that condition and invoke `ssl_context.load_cert_chain()` even if only `local_certificate_file` was provided.

**Comment by cannatag** [comment]
thanks! I check your PR and will include it in the next release of ldap3.

**Comment by cannatag** [comment]
included in ldap3 2.0.3. will be released soon. thanks
